src/index.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App/App';
-import { BrowserRouter } from 'react-router-dom';
-
+import { BrowserRouter } from 'react-router-dom'; 
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <BrowserRouter>
+  <BrowserRouter forceRefresh={true}>
```

Comment:
This makes the app refresh when the URL changes. It does cause some unwanted side effects with fetching. Basically, we are fetching more and can not store data. We may be able to fix this, but for now I think this is the best that we have.